src/main/java/cl/transbank/patpass/Options.java

```diff
@@ -0,0 +1,55 @@
+package cl.transbank.patpass;
+
+import cl.transbank.common.Option;
+import com.google.gson.annotations.SerializedName;
+import lombok.*;
+import lombok.experimental.Accessors;
+
+@ToString @NoArgsConstructor @AllArgsConstructor
+public class Options extends Option implements Cloneable  {
```

Comment:
This class could be named `PatpassOptions`

Reply:
done